_security/configuration/enabling-security.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Enabling security
+parent: Configuration
+nav_order: 45
+---
+
+# Enabling security
+
+The default version of OpenSearch comes with Security features pre-installed, however if the Security plugin was [disabled]({{site.url}}{{site.baseurl}}/security/configuration/disable/) or OpenSearch was installed without security, for example, using the minimal distribution method, you can enable the plugin as follows.
+
+A full cluster restart is necessary to enable security features.
+{: .label .label-red }
+
+## Installation steps for prod cluster
```

Comment:
I don't think we need an H2 if we only have one, so we can remove this line. You could replace it with a regular paragraph style and "To enable security:"

Reply:
This is updated now